janusgraph-core/src/main/java/org/janusgraph/core/JanusGraph.java

```diff
@@ -52,6 +52,14 @@
         test = "org.apache.tinkerpop.gremlin.process.computer.GraphComputerTest",
         method = "shouldProcessResultGraphNewWithPersistVertexProperties",
         reason = "The result graph should return an empty iterator when vertex.edges() or vertex.vertices() is called.")
+@Graph.OptOut(
+        test = "org.apache.tinkerpop.gremlin.structure.io.IoTest$GraphMLTest",
+        method = "shouldReadGraphMLWithNoEdgeLabels",
+        reason = "Titan does not support default edge label (edge) used when GraphML is missing edge labels.")
+@Graph.OptOut(
+        test = "org.apache.tinkerpop.gremlin.process.computer.GraphComputerTest",
+        method = "shouldSupportGraphFilter",
+        reason = "Titan currently does not support graph filters but does not throw proper exception because doing so breaks numerous tests in gremlin-test ProcessComputerSuite.")
```

Comment:
So the more correct statement here would be "graph filter support in JanusGraph is untested." FulgoraGraphComputer doesn't support graph filters, which is why the OptOut is needed. Note though I've recently used graph filters without issue on JanusGraph using SparkGraphCompter, for example.

If we were going to file an issue it would be that gremlin-test does not fully support computers that don't support graph filters. But I kind of think it's on JanusGraph ... update FulgoraGraphComputer to support graph filters (which I apologize I didn't do as part of this effort) or find a way to get rid of it (see discussion on this below).

https://github.com/JanusGraph/janusgraph/issues/41
https://groups.google.com/forum/#!topic/janusgraph-dev/rmdVpvIJOXc